node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -3218,6 +3349,13 @@ mod tests {
             .add_half_neighbor_key(existing_node_5_key.clone())
             .unwrap();
         root_node.resign();
+        root_node.metadata.last_update = dest_db.root().metadata.last_update;
+        root_node.inner.country_code = "AU".to_string();
+        root_node.metadata.node_location_opt = Some(NodeLocation {
+            country_code: "AU".to_string(),
+            free_world_bit: true,
+        });
```

Comment:
Again, I don't understand why the country specifications have to be specially treated.